src/libs/fileDownload/getImageManipulator/index.native.ts

```diff
@@ -1,13 +1,17 @@
-import {manipulateAsync} from 'expo-image-manipulator';
-import type {FileObject} from '@components/AttachmentModal';
+import {ImageManipulator} from 'expo-image-manipulator';
+import type {FileObject} from '@pages/media/AttachmentModalScreen/types';
 import type ImageManipulatorConfig from './type';
 
 export default function getImageManipulator({fileUri, width, height, type, fileName}: ImageManipulatorConfig): Promise<FileObject> {
-    return manipulateAsync(fileUri ?? '', [{resize: {width, height}}]).then((result) => ({
-        uri: result.uri,
-        width: result.width,
-        height: result.height,
-        type,
-        name: fileName,
-    }));
+    return ImageManipulator.manipulate(fileUri ?? '')
```

Comment:
Why do you need to add these changes here? We should add a QA steps to cover if behaviour did not change cc @parasharrajat did you test this please?